packages/core/src/utils/collect-updates/collect-updates.ts

```diff
@@ -44,7 +45,10 @@ export function collectUpdates(
       : new Map(filteredPackages.map(({ name }) => [name, packageGraph.get(name)]));
 
   let committish = commandOptions.since;
-  const tagPattern = isIndependent ? '*@*' : '';
+  let tagPattern = '';
+  if (isIndependent) {
+    tagPattern = describeTag ? describeTag : '*@*';
```

Comment:
so from what I see, this describe will only work for the `independent` mode and not the version mode, I wonder if it wouldn't be better to make it work for both mode?

Reply:
yes, it is only work for the `independent` mode, I'm not sure whether it will be better to make it work for both modes

Reply:
it's probably better to support it for both, I'm also wondering why the name `describe`? is this option only used for reading a tag list or will it also be used to add tag names? If it does add tag then I'm not sure of the word `describe`

Reply:
this option only used for reading a tag list, and now I have revised it to support for both.

Reply:
ok great then `describe` should be fine